src/app/dropdown/dropdown-angular-close-item-false.demo.html

```diff
@@ -5,12 +5,12 @@
   -->
 
 <div class="clr-example squeeze dropdown-demo">
-    <clr-dropdown [clrMenuPosition]="'bottom-right'" [clrCloseMenuOnItemClick]="false">
+    <clr-dropdown [clrCloseMenuOnItemClick]="false">
```

Comment:
Since we moved `clrMenuPosition` to the dropdown menu and renamed it as `clrPosition`, shouldn't we do the same for `clrCloseMenuOnItemClick` and rename it to `clrCloseOnItemClick`?